cmd/commands/common.go

```diff
@@ -282,23 +288,37 @@ func ensureGitToken(cmd *cobra.Command, cloneOpts *git.CloneOptions, verify bool
 
 func ensureGitPAT(cmd *cobra.Command, opts *RuntimeInstallOptions) error {
 	var err error
-	if !store.Get().Silent {
-		return getGitPATFromUserInput(cmd, opts)
+	tokenFromFlag, err := cmd.Flags().GetString("personal-git-token")
+	if err != nil {
+		return err
 	}
 
-	opts.GitIntegrationRegistrationOpts.Token, err = cmd.Flags().GetString("git-token")
-	return err
+	if  tokenFromFlag == "" {
+		if !store.Get().Silent {
+			err = getGitPATFromUserInput(cmd, opts)
+			if err != nil {
+				return err
+			}
+		} else {
+			opts.GitIntegrationRegistrationOpts.Token, err = cmd.Flags().GetString("git-token")
```

Comment:
same here, don't get the value like that, take it from `opts`

Reply:
Also, I kind of agree with noam here, this might be confusing, so lets add an info log here `Using runtime token as personal user token` or something like that.